internal/entity/model.go

```diff
@@ -0,0 +1,17 @@
+package entity
+
+type AddressField struct {
+	Name    string `json:"name"`
+	Address string `json:"address"`
+	Phone   string `json:"phone"`
+}
+
+func NewAddressField(name, address, phone string) (field *AddressField) {
+	field = new(AddressField)
+
+	field.Name = name
+	field.Address = address
+	field.Phone = phone
+
+	return
+}
```

Comment:
Probably, the code can be simplified:
```suggestion
func NewAddressField(name, address, phone string) (*AddressField) {

	return &AddressField{name, address, phone}
}
```